newDB.sql

```diff
@@ -1,97 +1,110 @@
-create table Branch (
-  IFS_Code varchar(255) not null primary key,
-  Branch_Name varchar(255) not null,
-  City varchar(255) not null,
-  Branch_Address varchar(255) not null,
-  State varchar(255) not null,
-  Country varchar(255) not null,
-  PIN_Code varchar(255) not null
+create table Branches (
```

Comment:
we follow snake case naming convention

Reply:
That was in previous branch I have changed it in new branch.